lib/components/src/blocks/PropsTable/PropJsDoc.tsx

```diff
@@ -7,24 +7,107 @@ interface PropJsDocProps {
   tags: JsDocTags;
 }
 
-const TBody = styled.tbody({ boxShadow: 'none !important' });
+export const Table = styled.table<{}>(({ theme }) => ({
```

Comment:
I think this is now outdated – can you merge `next` back into the branch to get the latest.

Reply:
Perfect